### DISCUSSION TITLE
Docker Hub images

### DISCUSSION THREAD

**Discussion by ferozsalam** [jertel/elastalert2#48] [Ideas]
Hi,

First off, thanks for your work on this project, ElastAlert has been in need of some forward momentum for some time. 

I have a small question - do you intend to start publishing tagged images of the `alt` versions to Docker Hub? Where I work we can only use fixed version tags rather than `latest` style tags, so this would be useful. I would be happy to help making this happen too, if that would be useful.

Thanks again.

**Comment by jertel** [comment]
I will tag the alt image with a formal release version next week. Thanks for the reminder.

**Reply by ferozsalam** [reply]
Just seen the new image, thanks very much!

**Comment by ferozsalam** [comment]
Hey @jertel could I trouble you for another tagged Docker image release please? There are some PRs in the last few weeks that we're eager to start making use of. Thank you!

**Reply by jertel** [reply]
Sure. There's a new latest-alt tagged image available today.

**Reply by ferozsalam** [reply]
We're only allowed to use specific version pins where I work, is there a chance you could push a `0.2.4-alt3` please?

**Reply by jertel** [reply]
The 0.2.4-alt3 tag is available as well now.

**Reply by ferozsalam** [reply]
Thank you!
